packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Sizes.Example.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { useConstCallback } from '@uifabric/react-hooks';
+
+// The panel type and description are passed in by the PanelSizesExample component (later in this file)
+const PanelExample: React.FunctionComponent<{ panelType: PanelType; description: string }> = props => {
+  const { description, panelType } = props;
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const openPanel = useConstCallback(() => setIsOpen(true));
+  const dismissPanel = useConstCallback(() => setIsOpen(false));
+
+  const a = 'aeiou'.indexOf(description[0]) === -1 ? 'a' : 'an'; // grammar...
+
+  return (
+    <div>
+      <DefaultButton text="Open panel" onClick={openPanel} />
+      <Panel
+        isOpen={isOpen}
+        onDismiss={dismissPanel}
+        type={panelType}
+        customWidth={panelType === PanelType.custom || panelType === PanelType.customNear ? '888px' : undefined}
+        closeButtonAriaLabel="Close"
+        headerText="Sample panel"
+      >
+        <p>
+          This is {a} <strong>{description}</strong> panel{panelType === PanelType.smallFixedFar ? ' (the default size)' : ''}.
+        </p>
+        <p>
+          Select this size using <code>{`type={PanelType.${PanelType[panelType]}}`}</code>.
```

Comment:
Do we usually show code visibly like this in examples?

Reply:
Not typically, but I thought it would be helpful in this case.

Reply:
i wonder if it makes sense to simply hide the code of some examples, especially when the code isn't the thing you'd want to show

Reply:
You may be talking about different things. I think Aneesha was referring to showing literal `PanelType.whatever` in the UI. @micahgodbolt I'm not totally sure what you're referring to (I have a guess but if you could elaborate that's better).

Reply:
oh i see what you mean :)